error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ClassRulesTestInput.java

```diff
@@ -23,4 +24,13 @@ Predicate<String> testClassReferenceIsInstancePredicate() throws IOException {
     Class<?> clazz = CharSequence.class;
     return s -> clazz.isInstance(s);
   }
+
+  Function<Number, Integer> testClassLiteralCast() {
```

Comment:
Curious why the pevious three tests threw exceptions as part of their signature 🤔

Reply:
Looks like a copy-paste error; will drop!